docs/backend-requests/testing/postman-or-insomnia.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Testing with Postman and Insomnia
+description: Learn about generating and using a long lived JWT Token with Postman or Insomnia.
+---
+
+# Testing with Postman and Insomnia
+
+Postman and Insomnia are powerful tools used to test API's, validate their behavior, and automate testing tasks. Basic testing with these tools is similar, and this guide will help you get started with either option.
+
+## Generate Long Lived JWT Template
+
+The standard token issued by Clerk expires after 60 seconds. Clerk SDKs handle refreshing the token regularly so that the authentication state is kept up to date. Because this token expires quickly, it isn't very useful when trying to test with Postman or Insomnia.
+
+You will want to create a long lived [JWT Template](/docs/backend-requests/making/jwt-templates) to be used in Postman or Insomnia. To do so, navigate to the [JWT Templates](https://dashboard.clerk.com/last-active?path=jwt-templates) page in the Clerk Dashboard. Click on the **New template** button and select the **Blank** template.
+
+<Images
+  src="/docs/images/testing/blank-jwt-template.webp"
+  width={3024}
+  height={1654}
+  alt="The JWT Template page in the Clerk Dashboard. The 'New template' button was clicked, in the modal that opened, the 'Blank' template is hovered over."
+/>
+
+Give your template a unique name, such as *'testing-template'*. Set the **Token Lifetime** to a value that suits your needs, or use the maximum of 315360000 seconds (10 years). If your normal token has Claims, then you can add the same Claims to your JWT Template.
+
+<Images
+  src="/docs/images/testing/testing-template.webp"
+  width={2492}
+  height={1400}
+  alt="Creating a JWT Template in the Clerk Dashboard"
+/>
+
+## Fetch Long Lived Token
+
+Visit your frontend that is using the same Clerk Application and instance that you want to test. Log in as a user. The user that you log in as will be the user you test with in Postman or Insomnia. You can create several tokens for several different users. Once you have logged in, open your Dev Tools and go to the **Console** tab. Enter the following command:
+
+```
+await window.Clerk.session.getToken()
+```
+
+<Images
+src="/docs/images/backend-requests/testing/get-token-from-console.png"
+width={1448}
+height={314}
+alt="Grab token from the console"
+/>
+
+## Using Postman or Insomnia
+
+Open Postman/Insomnia and create a new request. Configure the correct method and route. In the Authorization tab, select Bearer Token. Paste the token you copied from the console as the Bearer Token. Your request will now authenticate as the user you created the token with.
```

Comment:
"Configure the correct method and route" is vague. Are you talking about configuring something like for example, a GET request on https://domain.com/test ?
We can provide an example to make this part clearer. Even an image, if it feels right.

Again, for "In the Authorization tab, select Bearer Token", we should probably an image showing this as well in Postman and Insomnia. We can do something like a `<Tabs />` component, one for Postman and one for Insomnia, and then have screenshots to display what the copy is talking about. So then in each tab, we have an image showing how to configure a new API route test and an image for selecting Bearer token and pasting in the Bearer token.

Reply:
"Configure the correct method and route" is vague." I've rewritten this part.

There was an image showing the method, route and bearer token. I've revised and added options.

protected-route.png and protected-route2.png - Postman.
insomina-protected-route.png and insomina-protected-route2.png - Insomnia.

Note: I did not edit the <Image> tag with the new dimensions.